app/grandchallenge/archives/templates/archives/archive_user_groups_form.html

```diff
@@ -1,6 +1,10 @@
 {% extends "groups/groups_form_base.html" %}
 {% load url %}
 
+{% block title %}
+    Add user - {{ object.title }} - {{ block.super }}
```

Comment:
This is fine since this is only about making the title and breadcrumbs consistent, but for the next step, I think we should watch for capitalization. I notice that sometimes it's not capitalized, like  here, whereas in most places we do use caps.

Reply:
Absolutely, I have it in my notes. There are several places to fix capitalization.